components/ui/Toggle/Toggle.tsx

```diff
@@ -0,0 +1,43 @@
+import { cva, VariantProps } from "class-variance-authority";
+import { Switch } from "@headlessui/react";
+
+const toggleStyles = cva(
+  "relative inline-flex h-[38px] w-[74px] shrink-0 cursor-pointer rounded-full border-2 border-transparent transition-colors duration-200 ease-in-out focus:outline-none focus-visible:ring-2  focus-visible:ring-white focus-visible:ring-opacity-75",
```

Comment:
I mention this else where, but let's try to avoid using hardcoded px values. You may already know this but you can look at [this page](https://tailwindcss.com/docs/height) to see the builtin values.